api/v1alpha1/grafanaorganization_types.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// Finalizer needs to follow the format "domain name, a forward slash and the name of the finalizer"
+	// See https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#finalizers
+	GrafanaOrganizationFinalizer = "observability.giantswarm.io/grafanaorganization"
+)
+
+// GrafanaOrganizationSpec defines the desired state of GrafanaOrganization
+type GrafanaOrganizationSpec struct {
+	// DisplayName is the name displayed when viewing the organization in Grafana. It can be different from the actual org's name.
+	DisplayName string `json:"displayName"`
+
+	// Access rules defines user permissions for interacting with the organization in Grafana.
+	RBAC *RBAC `json:"rbac,omitempty"`
+}
+
+// RBAC defines the RoleBasedAccessControl configuration for the Grafana organization.
```

Comment:
```suggestion
// RBAC defines the RoleBasedAccessControl configuration for the Grafana organization.
// Each fields represents the mapping to a Grafana role:
//   Admin: full access to Grafana
//   Editor: edit resources in Grafana
//   Viewer: read only access to Grafana
// Each fields holds a list of string which represents values for a specific auth provider org attribute.
// The org attribute is looked up using the `org_attribute_path` which is configure on your Grafana instance, see `https://your-grafana-instance/admin/settings`.
// A user is granted a role when one of its org attribute value is contained within one of the role's values defined here.
// More info on Grafana org attribute and role mapping at [Configure role mapping](https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/#configure-role-mapping)
```

Reply:
Can you fix `which is configure` with `which is configured`?
Also, for the roles, it's limited to the organization so Admin: Full access inside the Grafana Organization

But I think this checks out :)

Reply:
edited

Reply:
Looks good to me <3

Reply:
We should also use 

`https://<YOUR_GRAFANA_INSTANCE>/admin/settings`  
to be really safe here